docs/src/pages/guides/FHIRSearchConfiguration.md

```diff
@@ -103,52 +100,81 @@ A few things to note are:
 
 In the preceding example, extension elements (on a Patient resource) with a url of `http://ibm.com/fhir/extension/Patient/favorite-color` are indexed by the `favorite-color` search parameter. To search for Patients with a favorite color of "pink", users could send an HTTP GET request to a URL like `[base]/api/v1/Patient?favorite-color:exact=pink`.
 
-For more information on search parameters, see the HL7 FHIR specification.
+For more information on search parameters, see the [HL7 FHIR specification](https://www.hl7.org/fhir/R4/searchparameter.html).
 
 #### 1.1.2 Recommendations
-When creating the SearchParameter FHIRPath expression, one should test the FHIRPath expression and test the search parameter.
+When creating the SearchParameter FHIRPath expression, be sure to test both the FHIRPath expression and the search parameter.
 
 If a search parameter expression extracts an element with a data type that is incompatible with the declared search parameter type, the server skips the value and logs a message. For choice elements, like Extension.value, its recommended to restrict the expression to values of the desired type by using the `as` function. For example, to select only Decimal values from the http://example.org/demical extension, use an expressions like Basic.extension.where(url='http://example.org/decimal').value.as(Decimal).
```

Comment:
Suggest back-tick "Basic.extension.where(url='http://example.org/decimal').value.as(Decimal)" so the hyperlink highlighting isn't weird.

Reply:
```suggestion
If a search parameter expression extracts an element with a data type that is incompatible with the declared search parameter type, the server skips the value and logs a message. For choice elements, like Extension.value, its recommended to restrict the expression to values of the desired type by using the `as` function. For example, to select only Decimal values from the http://example.org/demical extension, use an expressions like `Basic.extension.where(url='http://example.org/decimal').value.as(Decimal)`.
```